### ISSUE TITLE
A1111 Forge Lora don't shows up

### ISSUE THREAD

**Issue by Dudekpob** [tdrussell/diffusion-pipe#316]
Hello got problem SDXL Lora don't show up in A1111 Forge parameters 

# Ogólne ustawienia
output_dir = "runs/lora_training"  # Katalog wyjściowy dla wyników
save_every_n_epochs = 6  # Zapis co ile epok
epochs = 12 # Liczba epok
micro_batch_size_per_gpu = 2  # Rozmiar mikro-partii na GPU
gradient_accumulation_steps = 1  # Kroki akumulacji gradientu
#gradient_release=true
activation_checkpointing = true
#blocks_to_swap = 2  # np. 2 bloki na CPU, dostosuj do dostępnej pamięci CPU
# Ścieżka do konfiguracji datasetu
dataset = 'dataset.toml'

# Konfiguracja modelu
[model]
type = 'sdxl'
checkpoint_path = 'models/sdxl/illustriousXL_v01.safetensors'
dtype = 'bfloat16'
# You can train v-prediction models (e.g. NoobAI vpred) by setting this option.
#v_pred = true
# Min SNR is supported. Same meaning as sd-scripts
#min_snr_gamma = 5
# Debiased estimation loss is supported. Same meaning as sd-scripts.
#debiased_estimation_loss = true
# You can set separate learning rates for unet and text encoders. If one of these isn't set, the optimizer learning rate will apply.
unet_lr = 5e-4
text_encoder_1_lr = 1e-4
text_encoder_2_lr = 1e-4

# Konfiguracja adaptera LoRA
[adapter]
type = "lora"  # Typ adaptera (musi być "lora")
rank = 16  # Ranga LoRA (określa rozmiar adaptacji)
#dropout = 0.1  # Dropout w LoRA (opcjonalne, domyślnie 0.0)
dtype = "bfloat16"  # Typ danych dla LoRA (powinien być zgodny z modelem)
# init_from_existing = "path/to/pretrained/lora"  # Opcjonalne: ścieżka do istniejącego LoRA

# Ustawienia optymalizatora
#[optimizer]
#type = "adamw8bit"  # Typ optymalizatora (np. AdamW)
#lr = 0.0005  # Learning rate
#gradient_clipping = 1.0  # Ograniczenie gradientu

#
[optimizer]
type = 'adamw8bitkahan'
lr = 5e-6
betas = [0.9, 0.99]
weight_decay = 0.01
eps = 1e-8
#gradient_release = true

# Opcjonalne: ewaluacja
[eval_datasets]
# Można dodać listę datasetów ewaluacyjnych, np.:
# [[eval_datasets]]
# name = "eval1"
# config = "path/to/eval_dataset_config.toml"
#eval_every_n_epochs = 1  # Ewaluacja co ile epok
